src/builder.ts

```diff
@@ -9,7 +9,7 @@ import * as stream from 'stream';
 import * as util from 'util';
 import * as utils from './utils';
 import {Route, StaticRoute} from './router';
-import {Pod} from './pod';
+import Pod from './pod';
```

Comment:
We should follow the Google TS Style Guide which says "no default exports". Can you remove the default exports here?

https://google.github.io/styleguide/jsguide.html#es-module-exports

Reply:
Ok, I've removed all of the default exports.